console/src/components/Services/ApiExplorer/Actions.js

```diff
@@ -44,6 +44,18 @@ const CREATE_WEBSOCKET_CLIENT = 'ApiExplorer/CREATE_WEBSOCKET_CLIENT';
 const FOCUS_ROLE_HEADER = 'ApiExplorer/FOCUS_ROLE_HEADER';
 const UNFOCUS_ROLE_HEADER = 'ApiExplorer/UNFOCUS_ROLE_HEADER';
 
+const LOADING = 'ApiExplorer/LOADING';
```

Comment:
As this is the name of the action, could we rename it `SET_LOADING` or `TOGGLE_LOADING` or something like this to follow naming conventions we have in other places?